andela/healthchecks_tpot
# Ch integrates travis ci 135501283

#### What does this PR do?
Adds travis build status badge and coveralls test coverage badge.
#### Description of Task to be completed?
Setting up the Travis - CI badge and Coveralls badge
#### How should this be manually tested?
Open Github repo https://github.com/andela/healthchecks_tpot/ and switch to develop branch. You should be able to view the badges at the top of the ReadMe.
#### What are the relevant pivotal tracker stories?
#135501283
